Libs/MRML/Core/vtkMRMLNode.h

```diff
@@ -333,13 +333,42 @@ class VTK_MRML_EXPORT vtkMRMLNode : public vtkObject
   ///
   /// If set to NULL, multiple instances of this node class are allowed.
   ///
-  /// If set to a non-NULL string, the node will be a singletong and
+  /// If set to a non-NULL string, the node will be a singleton and
   /// the scene will replace this node instead of adding new instances.
   ///
   /// The SingletonTag is used by the scene to build a unique ID.
+  ///
+  /// If the there can only be one instance of a given node class in the scene,
+  /// then the singleton tag should be Singleton. For example, the interaction and
+  /// selection nodes are named Selection and Interaction, with Singleton tags set to
+  /// Singleton, and with IDs set to vtkMRMLSelectionNodeSingleton and
+  /// vtkMRMLInteractionNodeSingleton.
+  /// If the singleton node is associated with a specific module it should be
+  /// named for the module, which already needs to be unique. The Editor module
```

Comment:
How about this:
  /// If the singleton node is associated with a specific module it should use
  /// the module name, which already needs to be unique, as the tag. The Editor module